spec/service/Dockerfile

```diff
@@ -20,17 +20,14 @@ WORKDIR /root
 #arg is rewritten by docker build
 ARG EC_REV_V1_2BETA=v1.2beta.fukuoka.0
 
-#COPY ./*.yml ./
-#COPY ./agent.sh ./
+COPY ./config.yaml /root/config.yaml
 
 RUN apk add --no-cache git
 RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing hub
-RUN wget -O ./agent_linux_sys.tar.gz https://raw.githubusercontent.com/Enterprise-connect/sdk/${EC_REV_V1_2BETA}/dist/agent/agent_linux_sys.tar.gz \
+RUN wget -O ./agent_linux_sys.tar.gz https://raw.githubusercontent.com/EC-Release/sdk/${EC_REV_V1_2BETA}/dist/agent/agent_linux_sys.tar.gz \
```

Comment:
make it more controllable check this out https://github.com/EC-Release/oci/blob/v1_oci_spec/spec/agent/Dockerfile#L29

Reply:
@ayasuda2OO3 . Looks like we have to deploy new one for EC service v1.2beta version. Will work on it.

Reply:
Added script as suggested. But for now using v1.1beta until v1.2beta agent available